Makefile

```diff
@@ -130,7 +130,7 @@ endef
 COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
 .PHONY: $(COMMON_CONSTRAINTS_TXT)
 $(COMMON_CONSTRAINTS_TXT):
-	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
```

Comment:
Nit: `curl` has a `-O` which will do the same thing as the `wget -O` so maybe just use `curl -OL` here?

Ref: https://curl.se/docs/manpage.html#-O

Reply:
they're a bit different:

`wget -O "$(@)"` means "write to a file named `$(@)`", which means "write to the Makefile target", which is requirements/common_constraints.txt

`curl -O` means "write to a file of the same name as the remote file", which would be just common_constraints.txt (no requirements/ prefix)